src/components/MyAccount/Settings/AccountSettingsDisplayOptions.tsx

```diff
@@ -100,6 +144,8 @@ export const AccountSettingsForm = ({ patron }: { patron: Patron }) => {
               id="phone-text-input"
               labelText="Update phone number"
               showLabel={false}
+              onChange={handleInputChange}
+              placeholder="000-000-0000"
```

Comment:
I'm not seeing the place holder show up when I delete the input value. How are you seeing it?

Reply:
Hmm I am. When the text input is entirely cleared there's nothing?

Reply:
i didn't have the latest changes 🤦🏻‍♀️ it's working great!